Rubberduck.Core/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -566,7 +566,7 @@ private void ExecuteRemoveComand(object param)
 
         private bool CanExecuteExportAllCommand => ExportAllCommand.CanExecute(SelectedItem);
 
-        public Visibility ExportVisibility => CanExecuteExportAllCommand ? Visibility.Collapsed : Visibility.Visible;
+        public Visibility ExportVisibility => _vbeKind == VBEKind.Standalone || CanExecuteExportAllCommand ? Visibility.Collapsed : Visibility.Visible;
```

Comment:
this is already done in the CanExecute. Duplicating it in the ViewModel is not useful and it muddies responsibilities in determining executability

Reply:
The CanExecute already returns false - ExportVisibilty is just  the inverse of ExportAllVisibility (i.e. if one is visible, the other is collapsed). I agree it's suboptimal - ideally we would base it on ExportCommand.CanExecute, but that could change current behaviour.